### PR TITLE
Refactor py2 support

### PR DESCRIPTION
**Pull request by JsseL**
Improves the work started in #17.

### CODE REVIEW

File: database_sanitizer/__main__.py [hunk 1]
```diff
@@ -3,9 +3,13 @@
 from __future__ import unicode_literals
 
 import argparse
+import codecs
+import io
```
Comment (suutari-ai) [inline database_sanitizer/__main__.py]: Are you really using `io` somewhere?

File: database_sanitizer/dump/mysql.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 from __future__ import unicode_literals
 
+import codecs
 import io
```
Comment (suutari-ai) [inline database_sanitizer/dump/mysql.py]: Is it still using `io`?

File: database_sanitizer/dump/postgres.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 from __future__ import unicode_literals
 
+import codecs
 import io
```
Comment (suutari-ai) [inline database_sanitizer/dump/postgres.py]: Is it still using `io`?

**Comment by suutari-ai** [comment]
Looks very good. Only the `io` imports seem unneeded. (Why there isn't flake8 configured to complain about these?)

**Comment by JsseL** [comment]
Fixed according to the comments. Linters should be configured indeed.

**Pull request merged**